UI/HTML/Amend_admin_or_staff_user_acct.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="description" content="Banka">
+      <meta name="keywords" content="Bank For the people">
+      <meta name="author" content="Shalu Chandwani Vaswani">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>Banka | Amend Account Details </title>
+    <link rel="stylesheet" href="../css/style1.css">
+  </head>
+  <body>
+      <header>
+          <div  id="Header" class="container1">
+              <h1>Banka<p>Your Trust Our Commitment</p></h1>
+            </div>
+            <nav>
+              <ul>
+                <form method="get" action="../index.html">
+                  <button type="Logout" name="button_Logout" class="button_Logout"><span></span>Logout</button></form>
+              </ul>
+          </nav>
+      </header> 
+  </script>
+  <script src="../js/jquery.js"></script>
+  <script type="text/javascript">
+
+  function getUrlVars() {
+    var vars = {};
+    var parts = window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+        vars[key] = value;
+    });
+    return vars;
+}
+   var dynamicContent = getUrlVars()["AccountType"];
+$(document).ready(function() {
```

Comment:
I can still see inline javascript here.

Move all javascript into a seperate file.

Reply:
changed it